core/src/core-api/config.js

```diff
@@ -331,6 +338,39 @@ class LuigiConfig {
   getGlobalContext() {
     return this.config?.navigation?.globalContext || {};
   }
+
+  /**
+   * Updates the context values for visible iframes and LUI web components.
+   * Note: the updated context values are not persisted. The developers have to do it on their own.
+   * @param {Object} ctx - The context to be updated.
+   * @memberof Configuration
+   * @since NEXTRELEASE
+   */
+  updateContextValues(ctx) {
+    const visibleIframes = IframeHelpers.getMicrofrontendIframes();
+    if (visibleIframes && visibleIframes.length > 0) {
+      visibleIframes.forEach(iframe => {
+        if (iframe.luigi) {
```

Comment:
what is luigi in iframe.luigi , is it a node or some other data. would be nice to just have a one line very short comment above this if statement.